Podcast/InternalProfileViewController.swift

```diff
@@ -192,11 +193,21 @@ class InternalProfileViewController: ViewController, UITableViewDelegate, UITabl
             cell.selectionStyle = .gray
             return cell
         case subscriptionsTableView:
-            let cell = tableView.dequeueReusableCell(withIdentifier: reusableSubscriptionCellID, for: indexPath) as? SearchSeriesTableViewCell ?? SearchSeriesTableViewCell()
-            if let series = subscriptions?[indexPath.row] {
-                cell.configure(for: series, index: indexPath.row, showLastUpdatedText: true)
+            guard let subs = subscriptions else { return UITableViewCell() }
```

Comment:
should a `SearchSeriesTableViewCell` be returned here instead ?

Reply:
I don't want to return a SearchSeriesTableViewCell in the case that the subscriptions haven't been fetched yet (i.e. they are nil) because then there would just be a white cell